### PR TITLE
Security - Prevent remote environments creation with HTTP protocol

### PR DESCRIPTION
**Pull request by DrRaider**

### CODE REVIEW

File: services/prompter.js [hunk 1]
```diff
@@ -246,6 +246,10 @@ async function Prompter(program, requests) {
         name: 'appHostname',
```
Comment (arnaudbesnier) [inline services/prompter.js]: Where is the change description in the `CHANGELOG.md`? 😭

File: services/prompter.js [hunk 2]
```diff
@@ -246,6 +246,10 @@ async function Prompter(program, requests) {
...
         message: 'What\'s the IP/hostname on which your application will be running? ',
         default: 'localhost',
```
Comment (arnaudbesnier) [inline services/prompter.js]: Did you check the case if you set a remote environment with the `lumber generate` option (without prompt)?
Is the error well catched/displayed?

File: services/prompter.js [hunk 3]
```diff
@@ -246,6 +246,10 @@ async function Prompter(program, requests) {
...
+        validate: (hostname) => {
+          if (hostname && /^http((s:\/\/.*)|(s?:\/\/(localhost|127\.0\.0\.1).*))/i.test(hostname)) { return true; }
+          return 'Hostname must be a valid url, https is mandatory except for localhost and 127.0.0.1';
+        },
```
Comment (arnaudbesnier) [inline services/prompter.js]: I would split the validation with 2 specific error messages (URL format validation and protocol validation)

File: services/prompter.js [hunk 4]
```diff
@@ -246,6 +246,10 @@ async function Prompter(program, requests) {
         name: 'appHostname',
         message: 'What\'s the IP/hostname on which your application will be running? ',
         default: 'localhost',
+        validate: (hostname) => {
+          if (hostname && /^http((s:\/\/.*)|(s?:\/\/(localhost|127\.0\.0\.1).*))/i.test(hostname)) { return true; }
+          return 'Hostname must be a valid url, https is mandatory except for localhost and 127.0.0.1';
```
Comment (arnaudbesnier) [inline services/prompter.js]: [Wording]
"Application hostname must be a valid url."
"HTTPS protocol is mandatory, except for localhost and 127.0.0.1."

File: services/prompter.js (hunk 2 above)
Comment (DrRaider) [reply services/prompter.js]: No it's not, I have to fix this.

Comment (DrRaider) [reply services/prompter.js]: Done !

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Security - Prevent remote environments creation with HTTP protocol
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: [Conventions] Tiny detail :P 
```suggestion
- Security - Prevent remote environments creation with HTTP protocol.
```
(If we implement "conventional commits" I am pretty sure we wont have

**Pull request merged**